value.go

```diff
@@ -43,7 +43,7 @@ var (
 
 // ToValue will convert an interface{} value to a value digestible by otto/JavaScript
 //
-// This function will not work for advanced types (struct, map, slice/array, etc.) and
+// This function will not work for advanced types (struct, map, slice/array, func, etc.) and
```

Comment:
That comment is correct, the conversion requires an Otto instance. I added that note to the global ToValue functon to avoid confusion.